style.css

```diff
@@ -24,14 +24,24 @@ body {
   margin: 0 auto;
   border-radius: 100%;
   background-color: #555;
+  transition: all 250ms ease-out;
 }
 
 .active#red {
   background-color: red;
 }
 
 .active#yellow {
-  background-color: yellow;
+  background-color: #555555;
+  animation-duration: 1s;
+  animation-name: blink;
+  animation-iteration-count: 2;
+  animation-timing-function: ease-in;
+}
```

Comment:
Consider looking at how you can consolidate some (or all) of these animation properties into the single `animation:` property.